tests/common/factories/group_scope.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import random
+
+import factory
+
+from h import models
+
+from .base import ModelFactory
+from .group import Group
+
+
+class GroupScope(ModelFactory):
+    class Meta:
+        model = models.GroupScope
+
+    hostname = factory.Faker('domain_name')
```

Comment:
As requirements change, will the [`domain_name` provider](http://faker.readthedocs.io/en/master/providers/faker.providers.internet.html#) suffice? It might if validation is all outside of the model...

Reply:
I guess that even if we added validation inside the model (validating that it's a domain name) then this domain name provider should continue to suffice? Since I think it always returns valid domain names.